src/App.css

```diff
@@ -35,14 +35,17 @@ li {
 
 #addBook .container {
   display: grid;
-  grid-template-columns: 4fr 4fr 2fr;
+
+  /* grid-template-columns: 4fr 4fr 3fr 2fr; */
```

Comment:
- [ ] Please remove commented code from your codebase. It will make your code clean and people don't know it existed before. For finding the code, there are [tools and git commands](http://stackoverflow.com/q/278192/971592) to help you look at the history of a file. :+1: